src/feature.h

```diff
@@ -32,6 +32,11 @@ class Features {
 #include "src/feature.def"
 #undef WABT_FEATURE
 
+#define WABT_FEATURE(variable, flag, help) \
+  void variable##_enable() { variable##_enabled_ = true; }
```

Comment:
This would turn into `threads_enable()` which sounds a bit odd. Maybe `enable_threads()` instead?